src/OpenTelemetry.Instrumentation.AspNet/CHANGELOG.md

```diff
@@ -8,6 +8,29 @@
   * New overload of `AddAspNetInstrumentation` now accepts a configuration delegate.
   * The `Enrich` can be used to add additional metric attributes.
 
+* BREAKING: HTTP server metrics now follow stable
+  [semantic conventions](https://github.com/open-telemetry/semantic-conventions/blob/v1.23.0/docs/http/http-metrics.md#http-server)
+  ([#1429](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1429)).
+
+  * New metric: `http.server.request.duration`
+    * Unit: `s` (seconds)
+    * Histogram Buckets: `0.005, 0.01, 0.025, 0.05, 0.075, 0.1, 0.25, 0.5,
```

Comment:
Need to explicitly mention that this buckets are coming because OTel Sdk is hardwired to specially treat this metrics, and mention the starting version of OTel SDK as well. (once we know the version)

Reply:
Sounds good to me, but I'll hold off until we know.

Reply:
I've went ahead with the suggested changes.